dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportController.java

```diff
@@ -70,11 +71,13 @@
 {
     protected static final String ENROLLMENTS = "enrollments";
 
-    private static final String DEFAULT_FIELDS_PARAM = "*,!relationships,!events,!attributes";
+    private static final String DEFAULT_FIELDS_PARAM_STRING = "*,!relationships,!events,!attributes";
+
+    static final List<FieldPath> DEFAULT_FIELDS_PARAMS = FieldFilterParser.parse( DEFAULT_FIELDS_PARAM_STRING );
 
     private static final EnrollmentMapper ENROLLMENT_MAPPER = Mappers.getMapper( EnrollmentMapper.class );
 
-    private final EnrollmentCriteriaMapper enrollmentCriteriaMapper;
+    private final EnrollmentParamsMapper enrollmentParamsMapper;
```

Comment:
```suggestion
    private final EnrollmentParamsMapper paramsMapper;
```

is probably descriptive enough 😄 and you've used it in another controller already